spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -1,22 +1,32 @@
 require "rails_helper"
 
-RSpec.feature "user browses items" do
-  let(:create_item) { LoanRequest.create( ) }
+RSpec.feature "unauthenticated user browses loan requests" do
+  let(:create_item) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.